classes/class-sailthru-content.php

```diff
@@ -267,7 +297,7 @@ function generate_payload( $post, $post_id ) {
 		$data['author']            = get_the_author_meta( 'display_name', $post->post_author );
 		$data['date']              = $post->post_date;
 		$data['vars']['post_type'] = $post->post_type;
-		$data['spider']            = 1;
+		$data['spider']            = $spider_value=="true" ? 1 : 0;
 
 		if ( ! empty( $post->post_excerpt ) ) {
```

Comment:
suggestion(non-blocking): Can we add the 'description' key initialization to main $data initialization?

I think it would be good to initialize $data['description'] with the most common case, probably $post->post_exerpt (barring an exception wont be thrown trying to access the attributes of $post), that way all the memory for $data can get allocated at once. Also I think it will help reduce branching.